EIPS/eip-5827.md

```diff
@@ -225,6 +225,10 @@ Renewable allowances can be implemented with discrete resets per time cycle. How
 
 Existing EIP-20 token contracts can delegate allowance enforcement to a proxy contract that implements this specification.
 
+## Implementation
+
+An audited, open source implemention of this standard as a IERC5827Proxy is [Token Funnels](https://github.com/suberra/funnel-contracts)
```

Comment:
If you would like, you can include a _minimal_ reference implementation in the `../assets/eip-5827` directory and link to it there. You'll want to strip it down to just enough to cover the specification, and remove any build system files.

Enclose inline code in backticks to get `eipw` to shut up:

```markdown
`IERC5827Proxy`
```